kie-server-parent/kie-server-tests/kie-server-integ-tests-all/pom.xml

```diff
@@ -259,6 +259,7 @@
               <org.jbpm.server.ext.disabled>${org.jbpm.server.ext.disabled}</org.jbpm.server.ext.disabled>
               <org.drools.server.ext.disabled>${org.drools.server.ext.disabled}</org.drools.server.ext.disabled>
               <org.optaplanner.server.ext.disabled>true</org.optaplanner.server.ext.disabled>
+              <org.kie.prometheus.server.ext.disabled>false</org.kie.prometheus.server.ext.disabled>
```

Comment:
I think that "convention" here is that what extension is actually tested in a specific module is right at the top defined as false inside `<properties>` tag. And here it is only referenced via `${}`. But probably @sutaakar knows the best how this should be enabled.

Reply:
happy to change it if needed, let me know.

Reply:
I think it would be good to keep the same style everywhere. @cristianonicolai Please adjust this.

Reply:
done